python/kserve/kserve/protocol/rest/openai/config.py

```diff
@@ -42,9 +32,6 @@ def get_open_ai_models(repository: ModelRepository) -> List[Model]:
 
 
 def maybe_register_openai_endpoints(app: FastAPI, model_registry: ModelRepository):
-    # Check if the openai package is available before continuing so we don't run into any import errors
-    if not openai_is_available():
```

Comment:
We might still need this check otherwise openai endpoints get registered to other model serving runtimes?

Reply:
But we can not use the approach checking if openai dependency exists or not now